packages/terra-application-layout/src/terra-dev-site/doc/example/ApplicationLayoutExampleCommon.scss

```diff
@@ -0,0 +1,5 @@
+:local {
```

Comment:
Can we remove this file and just use `ApplicationLayoutCommon.scss`? They provide the same styles and class name

Reply:
Ah I see. One if for the tests.